ArduCopter/GCS_Mavlink.cpp

```diff
@@ -1476,4 +1486,15 @@ uint8_t GCS_MAVLINK_Copter::high_latency_wind_direction() const
     }
     return 0;
 }
+
+int8_t GCS_MAVLINK_Copter::high_latency_air_temperature() const
+{
+#if AP_AIRSPEED_ENABLED
+    return GCS_MAVLINK::high_latency_air_temperature();
+#else
+    return INT8_MIN;    // NOTE high latency doesn't specify invalids for fields
```

Comment:
@stephendade High_latency does not define what fields should be set to in case or invalid data. Any suggestions?

Reply:
I've used 0 to denote any case where to data is unvailable or otherwise invalid in the rest of high_latency, so best to be consistent with that.

Reply:
I'm not a fan of 0 for this case. As 0 deg C is certainly possible for air temperature. I'd say either `INT_MAX` or INT_MIN` makes the most sense?

Reply:
Go with ``INT_MIN``. I'll need to raise a PR to fix the rest of HL to make it consistent.